packages/rancher-vsphere-csi/charts/README.md

```diff
@@ -71,3 +71,18 @@ More information on managing Secrets using kubectl [here](https://kubernetes.io/
 ## Migration
 
 The CSI migration feature is only available for vSphere 7.0 U1.
+
+Follow the steps [here](https://rancher.com/docs/rancher/v2.6/en/cluster-provisioning/rke-clusters/cloud-providers/vsphere/out-of-tree/vsphere-volume-migration/?#) for CSI Migration.
+
+## Upgrade Kubernetes version after Migration
+
+- To upgrade the Kubernetes version after Migration Follow the [Upgrading Kubernetes version](https://rancher.com/docs/rancher/v2.5/en/cluster-admin/upgrading-kubernetes/) steps.
+
+- While the cluster is updating, if you have a workload with a bound volume attached, you would find 2n pods as before. The first set will be new and active and the second set will be stuck in **Terminating** state.
+
+- For these pods in **Terminating** state, delete that pod using `kubectl delete pod <pod_id> --force=true --timeout=120s --grace-period=0`
+  * **Note** you must wait for the pod(s) to become stuck in this state. If you remove pods before they have a working duplicate, `--grace-period=0` will remove any existing data including the bound volume.
+
+- The upgrade will continue after the pod(s) stuck in terminating stage are removed.
+
+- If there is more than 1 node where workloads that have bound volumes from the migration, this step must be repeated for each node
```

Comment:
Can we change this to a paragraph rather than bullet points since it all basically reads like a paragraph? 

This could also be simplified to something like this:

Follow the steps in the [Upgrading Kubernetes](https://rancher.com/docs/rancher/v2.5/en/cluster-admin/upgrading-kubernetes/) guide to upgrade the Kubernetes version after migration.

**Important:** If you have a workload with a bound volume attached to it during an upgrade, you will see the pods get redeployed and the old pods will get stuck in a **Terminating** state. You can safely delete the old stuck pods by executing the command `kubectl delete pod <pod_id> --force=true --timeout=120s --grace-period=0` to fix this problem, but it's imperative you verify before deleting a pod, that there is a new active pod corresponding to the old pod you want to delete. Deleting an old pod before a new one has been deployed could result in loss of data and bound volumes (Due to `--grace-period=0`).

Reply:
@PennyScissors I made the changes as you mentioned. Could you please review it again?